packages/react-components/react-shared-contexts/etc/react-shared-contexts.api.md

```diff
@@ -134,7 +134,6 @@ export const CustomStyleHooksContext_unstable: React_2.Context<Partial<{
     useMessageBarActionsStyles_unstable: CustomStyleHook;
     useMessageBarGroupStyles_unstable: CustomStyleHook;
     useToasterStyles_unstable: CustomStyleHook;
-    useTeachingPopoverStyles_unstable: CustomStyleHook;
```

Comment:
It's a breaking change, please revert. If these methods are not used, please mark them as deprecated.

Reply:
Do we need to be concerned about a breaking change in unstable prior to the actual functionality being added? This hasn't been consumed yet and was simply an oversight from scaffolding auto-gen, not something we intended to be merged.

Reply:
react-shared-contexts is not preview, so in this case removal is a breaking change that we need to avoid. We should mark it deprecated for now.

Reply:
Added back in - cheers for pointing this out, will be careful about these changes in the future.